kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/integration/GcpClientBuilderIT.java

```diff
@@ -18,7 +26,23 @@
 @Category(IntegrationTest.class)
 public class GcpClientBuilderIT extends BaseConnectorIT {
 
-    private BigQuerySinkConfig connectorProps(GcpClientBuilder.KeySource keySource) throws IOException {
+    Logger logger= LoggerFactory.getLogger(GcpClientBuilderIT.class);
+    private BigQuery bigQueryInstance;
+
+    private TableId tableId;
+    @Before
+    public void setup() throws Exception {
+        bigQueryInstance = newBigQuery();
+        tableId = TableId.of(project(),dataset(),"authenticate-storage-api");
+        if(bigQueryInstance.getTable(tableId) == null) {
+            logger.info("Going to Create table : "+ tableId.toString());
+            bigQueryInstance.create(TableInfo.of(tableId, StandardTableDefinition.newBuilder().build()));
+            logger.info("Created table : "+ tableId.toString());
+            // table takes time after creation before being available for operations. You may have to wait a few minutes (~5 minutes)

```

Comment:
I do not see any wait command here. Are we waiting at some other place?

Reply:
No, I have manually created the table for IT. If later on we modify the table name or delete the existing one, a subsequent run should be able to succeed. The stream writer fails with sort of a generic error which does not help tell if it is a missing table or un-authenticated user. So, keeping the table pre-created. 

Also I have written a method [here](https://github.com/confluentinc/kafka-connect-bigquery/pull/302/files#diff-367b30d794bf73d208897cc477ab7aa565616a2d9d02b2ad4ecdb72c65c04344R34) and I am planning to modify this integration test later to use this method instead as this will handle table creation as well.